### ISSUE TITLE
Tasks don't contribute to Goals anymore, in v2024.11.17.570-fdroid

### ISSUE THREAD

**Issue by Katsumi-III** [Medvedev91/timeto.me#85]
I used to use the goal feature to track my two types of language learning (and other activities) immersion, with the activity name being what I'm watching or analyzing. For example, I'm going to use my "flow immersion" activity I would input "【Core Keeper】なんと今回はスペシャルゲストの方々とコアキーパー✨【ぶいすぽっ！/英リサ】" , which is a YouTube video title, on tasks, then tap it, then select my flow immersion named activity. Throughout my day, I would use this process and watch my goal bars progress until I reach my goal. This process was quite motivating for my use case. If this is not a bug but a feature, then I'll go back to the previous version. Perhaps my use case is not what the dev of this app intended. Nevertheless, thank you for reading my complaint? Issue?

**Comment by Medvedev91** [comment]
Hi! I'm not sure I understood you right, but if you need the old behaviour, just set the "note" for the goal empty.

**Comment by Katsumi-III** [comment]
Sorry for not putting enough time in properly explaining my issue. Let me try again:
 This is my usage of your application, if it happens to not align with your intended usage then I'll close the issue so as to not waste your time.

This is a screenshot of me reverted back to the previous version. My use case is shown with that task called "[#VRChat] [参加型] 体感するVRアー...". This tasks is using the "flow immersion" activity. If I start that task then the goal with the word "flow" at the end would progress. 

With the latest version, this does not happen. Of course, I could just create the goal and just tap the new goal and start my "flow immersion" activity, there the goal "動画の見 flow" would progress, but, I like keeping tabs of what Japanese media I used during these periods. 

With the new update I am unable to record the media I used through the tasks I created and titled them the title of the Japanese media I used for dialogue analysis or just recording what I'm watching. 
![Screenshot_20241123_161444](https://github.com/user-attachments/assets/9bbea684-171e-453a-9062-b97420015c2f)
By the way, I tend to accumulate a bunch of paused tasks because I want to come back to them later, but then I pause that task and start another, repeat process and now I have a giant list of paused tasks. If there's anything that would help delete them all then I would appreciate it. Anyway, thank you for making this application and open sourcing it. This app helped me be much more aware on how I spend my time, and forces me to spend it properly.

**Comment by Katsumi-III** [comment]
I also use the goals as a tracker of certain activities. Like the goal "eating or/and Rest" and "アンキの復習" to make sure how I spend my time on those activities, shown right in my face, in the main section of the app I'm in. Un-ignorable.

**Comment by Medvedev91** [comment]
Okay, before we continue, just to clarify. When you create a goal "動画の見 flow" what "note" did you set? Left blank or entered "動画の見 flow"?

**Comment by Medvedev91** [comment]
> This tasks is using the "flow immersion" activity. If I start that task then the goal with the word "動画の見 flow" would progress.

Hmm.. In the previous version the goal bar showed the activity name. Isn't it? If the task uses the "flow immersion" activity, so the bar with the text "flow immersion" should progres.

**Comment by Katsumi-III** [comment]
> > This tasks is using the "flow immersion" activity. If I start that task then the goal with the word "動画の見 flow" would progress.
> 
> Hmm.. In the previous version the goal bar showed the activity name. Isn't it? If the task uses the "flow immersion" activity, so the bar with the text "flow immersion" should progres.

Yes, my mistake on that part by calling the activity different than the goal, both activity and goal are called "動画の見 flow" .

**Comment by Medvedev91** [comment]
What about this question?

> Okay, before we continue, just to clarify. When you create a goal "動画の見 flow" what "note" did you set? Left blank or entered "動画の見 flow"?

Did you try not to fill the "note" field here? Like this:

![image](https://github.com/user-attachments/assets/1dbfa291-be85-43ae-a82f-351d251d6a86)

**Comment by Katsumi-III** [comment]
When I created a goal for that activity, "動画の見 flow", I had added a note, and then tried without a note.
Since I wanted to track my activity usage through goals while keeping tabs on what I watched, I thought if tasks with the 動画の見 flow activity would contribute to the goal I created for the 動画の見 flow activity.

**Comment by Katsumi-III** [comment]
From what you described, yes. Though, did I misremember? Is this already working on the latest update? Just be absolutely sure, if I created a blank goal of the Work activity, and then I go on and create several tasks of the work activity, like: 1# resolving work issue,  2# Creating analysis diagrams, then both 1# and 2# will make the blank goal from the Work activity progress?

If this is already suppose to be working then maybe I made a mistake while checking the blank goal from my 動画の見 flow activity. Perhaps, while testing that, I haven't given enough time for the goal to update? I'll update mine and try again.

**Comment by Medvedev91** [comment]
On the screenshot the 569 version. Blank note means the goal collects all activity related tasks.

**Comment by Katsumi-III** [comment]
holy shit sorry. It seems like it really was the case. The blank goal had progressed. I think, at the time, I was under the belief that even if I hadn't waited for a full minute, the blank goal would of updated anyway somehow... I don't remember why I thought that.

Sorry for wasting your time like this! I did not mean to cause unnecessary confused head-scratches!

**Comment by Medvedev91** [comment]
Did you manage to get what you wanted?

**Comment by Katsumi-III** [comment]
YES! so sorry!

**Comment by Medvedev91** [comment]
> YES! so sorry!

Oh, okay! Don't worry.

**Comment by Katsumi-III** [comment]
thank you again for the app *bow* I really like it!

**Comment by Katsumi-III** [comment]
If you want me to test any features my discord is katsumi_bakeneko

**Comment by Medvedev91** [comment]
Don't forget to close the issue if it solved : )
